SPECS/apr-util/apr-util.spec

```
@@ -2,9 +2,9 @@
 
 Summary:        The Apache Portable Runtime Utility Library
 Name:           apr-util
-Version:        1.6.1
-Release:        6%{?dist}
-License:        ASL 2.0
+Version:        1.6.3
+Release:        1%{?dist}
+License:        Apache License 2.0
```

See: https://github.com/microsoft/CBL-Mariner/pull/4786/files#r1099232068.

```suggestion
License:        Apache-2.0
```